src/database/migrations/2021_01_04_221106_create_course_characteristics_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCourseCharacteristicsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('course_characteristics', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('code');
+            $table->unsignedTinyInteger();
```

Comment:
What does it mean? Why there is no "column name"?

Reply:
There isn't column "name" because there is "title" column